### PR TITLE
Force tty on ssh commands

### PR DESCRIPTION
**Pull request by grisu48**
Force the usage of tty (`-t`) for ssh commands on publiccloud instances, which is necessary for e.g. sudo and should not have any negative effects on other commands.

- Related ticket: https://progress.opensuse.org/issues/117271
- Verification run: https://duck-norris.qam.suse.de/tests/10862 | https://duck-norris.qam.suse.de/tests/10863

### CODE REVIEW

File: lib/publiccloud/instance.pm [hunk 1]
```diff
@@ -388,7 +388,7 @@ sub softreboot
         ssh_interactive_leave();
     }
 
-    $self->run_ssh_command(cmd => 'sudo shutdown -r +1');
+    $self->ssh_script_run(cmd => 'sudo shutdown -r +1');
```
Comment (asmorodskyi) [inline lib/publiccloud/instance.pm:391]: just to keep things clear - this is unrelated , right ? ( it is fine just for general understanding )

**Review by mimi1vx — APPROVED** [review]

File: lib/publiccloud/instance.pm (hunk 1 above)
Comment (grisu48) [reply lib/publiccloud/instance.pm:391]: This is required so that we use the modified `_prepare_ssh_cmd` subroutine
